sdk/communication/azure-communication-identity/tests.yml

```diff
@@ -5,18 +5,13 @@ extends:
   parameters:
     PackageName: azure-communication-identity
     SafeName: azurecommunicationidentity
-    Clouds: 'Public,PPE,Int'
+    Clouds: 'Public'
+    UseFederatedAuth: true
     CloudConfig:
       Public:
+        ServiceConnection: azure-sdk-tests
+        SubscriptionConfigurationFilePaths:
+          - eng/common/TestResources/sub-config/AzurePublicMsft.json
         SubscriptionConfigurations:
-          - $(sub-config-azure-cloud-test-resources)
           - $(sub-config-communication-services-cloud-test-resources-common)
```

Comment:
Are there still unique configurations in these configs that are still needed? I see some others like callautomation which don't need these configurations.

Reply:
The others are extending a common template: sdk/communication/communication-tests-template.yml where UseFederatedAuth is defaulted to false.

I had to override the flag as true in each of the test yml

Reply:
Is there a reason to not flip that shared template default to true?

Reply:
Just to be on the safe side of unneccessarily using federated auth.